components/Main/Experience.vue

```diff
@@ -0,0 +1,134 @@
+<script setup lang="ts">
+const featuresTitle = ref<HTMLElement | null>(null);
+
+const items: { title: string }[] = [
+  {
+    title: 'Recepción de pagos por tarjeta o transferencias',
+  },
+  {
+    title: 'Lectura de ubicación por GPS'
+  },
+  {
+    title: 'Chat conversacional'
+  },
+  {
+    title: 'Agendamiento de citas'
+  },
+  {
+    title: 'Almacenamiento en Base de datos'
+  },
+];
+
+function handleScroll() {
```

Comment:
This function should return void, please check the ero component
Are we using js or ts?

Reply:
<img width="255" alt="Captura de pantalla 2024-08-21 a la(s) 12 08 05 a  m" src="https://github.com/user-attachments/assets/001e5dc8-32cc-4e0f-9452-04ed74566570">

ts infers what returns the function, because is simple